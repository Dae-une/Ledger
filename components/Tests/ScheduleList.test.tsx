// import React from 'react';
// import { screen, render } from '@testing-library/react';
// import { Route } from 'react-router-dom';
// import { withRouter } from '../../tests/utils';
// import { scheduleData } from '../../mocks/data';
// import ScheduleList from '../ScheduleList/ScheduleList';

// describe('ScheduleList', () => {
//   it('render correctly', async () => {
//     const { asFragment } = render(withRouter(<Route path="/" element={<ScheduleList list={scheduleData} />} />));

//     expect(asFragment()).toMatchSnapshot();
//   });

//   it('리스트가 없으면 Empty화면을 보여준다.', () => {
//     render(withRouter(<Route path="/" element={<ScheduleList list={[]} />} />));

//     expect(screen.getByText('일정이 없어요.'));
//   });
// });
