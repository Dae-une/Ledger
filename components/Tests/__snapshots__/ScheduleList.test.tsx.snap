// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleList render correctly 1`] = `
<DocumentFragment>
  <div
    class="css-rjennl"
  >
    <div
      class="css-172fik6"
    >
      <div>
        제목
      </div>
      <div>
        내용
      </div>
    </div>
    <div
      class="css-172fik6"
    >
      <div>
        제목
      </div>
      <div>
        무슨일인지 쥬르르르르르르르르륵
      </div>
    </div>
    <div
      class="css-172fik6"
    >
      <div>
        제목2
      </div>
      <div>
        무슨일인지 쥬르르르르르르르르륵2
      </div>
    </div>
    <div
      class="css-172fik6"
    >
      <div>
        제목3
      </div>
      <div>
        무슨일인지 쥬르르르르르르르르륵3
      </div>
    </div>
    <div
      class="css-172fik6"
    >
      <div>
        제목4
      </div>
      <div>
        무슨일인지 쥬르르르르르르르르륵2
      </div>
    </div>
    <div
      class="css-172fik6"
    >
      <div>
        제목5
      </div>
      <div>
        무슨일인지 쥬르르르르르르르르륵1
      </div>
    </div>
    <div
      class="css-172fik6"
    >
      <div>
        제목
      </div>
      <div>
        무슨일인지 쥬르르르르르르르르륵
      </div>
    </div>
    <div
      class="css-172fik6"
    >
      <div>
        제목5
      </div>
      <div>
        무슨일인지 쥬르르르르르르르르륵1
      </div>
    </div>
    <div
      class="css-172fik6"
    >
      <div>
        제목
      </div>
      <div>
        무슨일인지 쥬르르르르르르르르륵
      </div>
    </div>
  </div>
</DocumentFragment>
`;
